Extension/src/packageVersion.ts

```diff
@@ -26,7 +26,13 @@ export class PackageVersion {
             if (firstDigitOffset !== -1) {
                 this.suffix = tokens[3].substring(0, firstDigitOffset);
                 this.suffixVersion = parseInt(tokens[3].substring(firstDigitOffset));
+            } else {
+                this.suffix = tokens[3];
+                this.suffixVersion = 0;
```

Comment:
suffixVersion should be 1, right?

Reply:
Yeah, definitely. Whether it's 1 or 0 will not affect version comparison (as we only apply suffices with a number of 2 or higher), but it's best to be technically correct in case we need to depend on that logic in the future.